packages/debugger/src/service.ts

```diff
@@ -859,6 +950,7 @@ export class DebuggerService implements IDebugger, IDisposable {
   private _sessionChanged = new Signal<IDebugger, IDebugger.ISession | null>(
     this
   );
+  private _stateRestored = new Signal<IDebugger, null>(this);
```

Comment:
It looks like `stateRestored` signal is not read in the final version. So let's get ride of it.

```suggestion
```